app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    result = split_integer(10, 2)
+
+    assert sum(result) == 10
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    result = split_integer(10, 2)
+
+    assert result == [5, 5]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    result = split_integer(3, 1)
+
+    assert result == [3]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    result = split_integer(10, 3)
+
+    assert result == [3, 3, 4]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts():
-    pass
+    result = split_integer(5, 6)
+
+    assert result == [0, 1, 1, 1, 1, 1]
```

Comment:
You don't need to introduce a variable for result. It's easier to read if you don't.

Reply:
Thanks!